requirements.txt

```diff
@@ -1,3 +1,3 @@
 fastapi==0.61.1
-jwt==1.0.0
+PyJWT==1.7.1
```

Comment:
https://pypi.org/project/PyJWT/


I have confirmed that the library string is correct in `PyJWT`.
However, version specification, I think it is better to **limit the minor version** because release timing.

Ex. `PyJWT ~= 1.7`

Reply:
recommendation to install
```
python-multipart==0.0.5
```